tests/integration/data/test_zone_align.json

```diff
@@ -0,0 +1,212 @@
+      "id": 1
+    },
+    {
+      "displayName": "AAAA",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone No Align",
+      "imageURL": "",
+      "id": 2
+    },
+    {
+      "displayName": "BBBB",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Invalid Align",
+      "imageURL": "",
+      "id": 3
+    },
+    {
+      "displayName": "BBBB",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Invalid Align",
+      "imageURL": "",
+      "id": 4
+    },
+    {
+      "displayName": "BBBB",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Invalid Align",
+      "imageURL": "",
+      "id": 5
+    },
+    {
+      "displayName": "CCCC",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Left Align",
+      "imageURL": "",
+      "id": 6
+    },
+    {
+      "displayName": "CCCC",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Left Align",
+      "imageURL": "",
+      "id": 7
+    },
+    {
+      "displayName": "CCCC",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Left Align",
+      "imageURL": "",
+      "id": 8
+    },
+    {
+      "displayName": "DDDD",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Right Align",
+      "imageURL": "",
+      "id": 9
+    },
+    {
+      "displayName": "DDDD",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Right Align",
+      "imageURL": "",
+      "id":10 
+    },
+    {
+      "displayName": "DDDD",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Right Align",
+      "imageURL": "",
+      "id": 11 
+    },
+    {
+      "displayName": "EEEE",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Center Align",
+      "imageURL": "",
+      "id": 12 
+    },
+    {
+      "displayName": "EEEE",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Center Align",
+      "imageURL": "",
+      "id": 13 
+    },
+    {
+      "displayName": "EEEE",
+      "feedback": {
+        "incorrect": "No",
+        "correct": "Yes"
+      },
+      "zone": "Zone Center Align",
+      "imageURL": "",
+      "id": 14 
+    },
+  ],
+  "feedback": {
+    "start": "Intro",
+    "finish": "Final"
+  },
+  "targetImg": "data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI4MDAiIGhlaWdodD0iNjAwIiBzdHlsZT0iYmFja2dyb3VuZDogI2VlZjsiPjwvc3ZnPg==",
+  "targetImgDescription": "This describes the target image"
```

Comment:
@pomegranited I think `targetImg` and `targetImgDescription` would be OK to omit for the purpose of testing alignment.